### DISCUSSION TITLE
so-web.service unit could not be found

### DISCUSSION THREAD

**Discussion by KennethBourque** [Security-Onion-Solutions/securityonion#13905] [2.4]
### Version

2.4.110

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Import

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

24

### RAM

64 GB

### Storage for /

1 TB

### Storage for /nsm

200 GB

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I am currently a student pursuing my bachelors in cybersecurity. The course requires that I utilize security onion to follow along with the material. I have installed SO on virtualbox and then on hyper-v. Both installations do not allow me to access the web interface the farthest I have gotten is to find out the so-web.service can not be found. I am very new to this and after spending nearly 100 hours over the past month I have not been able to find a resolution. 

PLEASE HELP

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by AfroThundr3007730** [comment]
Pretty much all of the services are handled by containers, so you won't find them in the system configuration on the host itself. You should verify the web interface is listening and you'll want to check that the firewall allows your IP to connect.

I recommend reading through the documentation for more details on this: https://docs.securityonion.net/en/2.4/

**Reply by KennethBourque** [reply]
So the web interface is listening and I used the command:

sudo so-firewall includehost analyst <IP ADDRESS> as recommended in the documentation.

**Reply by cm-ops** [reply]
What is your IP and what does this return from the CLI - `sudo salt-call pillar.get firewall:hostgroups:analyst`?

How are you trying to access the SOC web page, by IP or hostname?
